MAUI/ProgressBar/getting-started.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Getting started with .NET MAUI ProgressBar control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI ProgressBar (Progress Bar) control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Getting started with .NET MAUI ProgressBar (Progress Bar)
+
+This section explains the steps required to add the progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill and track fill. This section covers only basic features needed to know to get started with Syncfusion progress bar.
+
+## Creating an application using the .NET MAUI ProgressBar
+
+* Create a new .NET MAUI application in the Visual Studio.
+
+* Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add ProgressBar to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.ProgressBar] then install that.
+
+### Register the handler
+
+Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```

Comment:
The Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for the Syncfusion core.

Reply:
done